modules/bbp-experiment/src/main/resources/schemas/bbp/experiment/patchedcell/v0.1.0.json

```diff
@@ -0,0 +1,197 @@
+              "node": "bbpqsh:QuantitativeValueShape",
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:startMembranePotential",
+              "name": "Start membrane potential",
+              "description": "Membrane potential of patched cell at beginning",
+              "node": "bbpqsh:QuantitativeValueShape",
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:endMembranePotential",
+              "name": "End membrane potential",
+              "description": "Membrane potential of patched cell at the end",
+              "node": "bbpqsh:QuantitativeValueShape",
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:sealResistance",
+              "name": "Seal resistance",
+              "description": "Seal resistance of patch",
+              "node": "bbpqsh:QuantitativeValueShape",
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:pipetteResistance",
+              "name": "Pipette resistance",
+              "description": "Resistance of pipette used",
+              "node": "bbpqsh:QuantitativeValueShape",
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:inputResistance",
+              "name": "Input resistance",
+              "description": "Input resistance of patched cell",
+              "node": "bbpqsh:QuantitativeValueShape",
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:seriesResistance",
+              "name": "Series resistance",
+              "description": "Series resistance while patching cell",
+              "node": "bbpqsh:QuantitativeValueShape",
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:liquidJunctionPotential",
+              "name": "Liquid junction potential",
+              "description": "Liquid junction potential of patch",
+              "node": "bbpqsh:QuantitativeValueShape",
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:analogToDigitalConverter",
+              "name": "Analog-to-digital converter settings",
+              "description": "Settings of analog-to-digital converter",
+              "node": "this:AnalogToDigitalConverterShape",
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:digitalToAnalogConverter",
+              "name": "Digital-to-analog converter settings",
+              "description": "Settings of digital-to-analog converter",
+              "node": "this:DigitalToAnalogConverterShape",
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:labelingCompound",
+              "name": "Labeling compound",
+              "description": "The compound used in the pipette to label the cell",
+              "node": "bbptlotsh:BrainRegionOntologyTermShape",
```

Comment:
Should the value of  "nsg:labelingCompound" be a brain region ?

Reply:
No, the value should be an ontology term of a labeling compound

Reply:
Excellent.